bundles/core/src/test/resources/list/test-content.json

```diff
@@ -77,6 +77,21 @@
                 "/content/list/pages/page_2"
             ]
         },
+        "staticOrderByTitleListTypeWithAccent": {
+            "jcr:primaryType"     : "nt:unstructured",
+            "sling:resourceType"  : "core/wcm/components/list",
+            "listFrom"            : "static",
+            "showThumbnail"       : "true",
+            "linkItems"           : "true",
+            "showDescription"     : "true",
+            "showModificationDate": "true",
+            "orderBy"             : "title",
+            "pages"               : [
+                "/content/list/pages/page_1",
+                "/content/list/pages/page_5",
```

Comment:
Just nitpicking but I would put `page_5` last here, makes it more clear that the order defined here is not the expected one.

Reply:
Done